Source/KerbalCombatSystems/DecouplerDesignator.cs

```diff
@@ -2,17 +2,17 @@
 {
     public class ModuleDecouplerDesignate : PartModule
     {
-        const string DecouplerDesignationGroupName = "Decoupler Designation";
+        const string DecouplerDesignationGroupName = "Seperator Designation";
 
         [KSPField(
             isPersistant = true,
             guiActive = true,
             guiActiveEditor = true,
-            guiName = "Decoupler Type",
+            guiName = "Seperator Type",
             groupName = DecouplerDesignationGroupName,
             groupDisplayName = DecouplerDesignationGroupName),
             UI_ChooseOption(controlEnabled = true, affectSymCounterparts = UI_Scene.None,
-            options = new string[] { "Default", "Weapon", "Countermeasure", "Escape Pod", "Warhead" })]
-        public string DecouplerType = "Default";
+            options = new string[] { "Default","Escape Pod", "Countermeasure", "Warhead" })]
```

Comment:
```suggestion
            options = new string[] { "Default", "Escape Pod", "Warhead" })]
```
Could we remove countermeasures here? Do you have plans to add them? In the mean time it could be confused to be related to interceptors.

Reply:
Will remove for time being, plans to include them for features like reactive armor or lore chaff